scripts/gha/get_pr_review_comments.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Fetches and formats review comments from a GitHub Pull Request."""
+
+import argparse
+import os
+import sys
+import firebase_github # Assumes firebase_github.py is in the same directory or python path
+
+# Attempt to configure logging for firebase_github if absl is available
+try:
+    from absl import logging as absl_logging
+    # Set verbosity for absl logging if you want to see logs from firebase_github
+    # absl_logging.set_verbosity(absl_logging.INFO)
+except ImportError:
+    # If absl is not used, standard logging can be configured if needed
+    # import logging as std_logging
+    # std_logging.basicConfig(level=std_logging.INFO)
+    pass # firebase_github.py uses absl.logging.info, so this won't redirect.
+
+
+def main():
+    # Default owner and repo from firebase_github, ensuring it's loaded.
+    default_owner = firebase_github.OWNER
+    default_repo = firebase_github.REPO
+
+    parser = argparse.ArgumentParser(
+        description="Fetch review comments from a GitHub PR and format for use with Jules.",
+        formatter_class=argparse.RawTextHelpFormatter # To preserve formatting in help text
+    )
+    parser.add_argument(
+        "--pull_number",
```

Comment:
Do you think it would be better to make this shorter, like --pr, or do you think --pull_number is best since it matches the GitHub API?